packages/ui/src/components/userProfile/userProfile.styled.js

```diff
@@ -0,0 +1,12 @@
+import styled from 'styled-components';
+import { Button } from 'antd';
+import { colors } from '../../../src';
```

Comment:
Don't import colors from `src` folder, it may cause circular dependencies issue. Please import from the colors file directly.

```javascript
import * as colors from '../../colors';
```

Reply:
I didn't notice it for I was using auto completion